etrago/cluster/networkclustering.py

```diff
@@ -985,8 +985,9 @@ def normed(x):
         generator_strategies=strategies_generators(),
         aggregate_one_ports=aggregate_one_ports,
         line_length_factor=kmean_settings["line_length_factor"],
+        scale_link_capital_costs=False
```

Comment:
Sorry for asking again, but do you think this change makes sense for all link carrier? Looking at PyPSA, this turns off the scaling of capital costs to the clustered link length. Is that correct? 
Setting it to False for links without a 'real length' like e.g. power_to_H2 makes sense for me. But for links with a real length e.G. DC-lines, the new length should be considered in the capital_costs. What do you think?